src/instruments/src/Common/gauges.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { FC, memo } from 'react';
+
+/**
+ * Calculates the rotation needed to position an item at a certain value mark on a half-circle gauge
+ *
+ * @param value     the desired value
+ * @param scaleMin  the lower bound of the half-circle gauge. Defaults to 0
+ * @param scaleMax  the upper bound of the half-circle gauge.
+ */
+function gaugeRotationFor(value, scaleMin = 0, scaleMax) {
+    return -90 + ((Math.min(scaleMax, Number(value)) - scaleMin) / (scaleMax - scaleMin)) * 180;
+}
+
+export type NeedleProps = {
+    x: number,
+    y: number,
+    length: number,
+    value: string | number,
+    scaleMin?: number,
+    scaleMax: number,
+    className?: string,
+    dashOffset?: number
+}
+
+export const Needle: FC<NeedleProps> = memo(({ x, y, value, length, dashOffset, scaleMin = 0, scaleMax, className }) => (
+    <path
+        className={`MidGrey GreyFill ${className}`}
```

Comment:
**thought:** Those colors are not used on any gauge in EIS2 as far as I know.

**suggestion:** Let's switch to a more sane default style or force providing `className`.